content/docs/lists-and-keys.md

```diff
@@ -1,30 +1,33 @@
 ---
 id: lists-and-keys
-title: Lists and Keys
+title: Lists dan Key
 permalink: docs/lists-and-keys.html
 prev: conditional-rendering.html
 next: forms.html
 ---
 
-First, let's review how you transform lists in JavaScript.
+Pertama-tama, mari kita tinjau kembali bagaimana anda dapat mentransformasi list di JavaScript.
 
-Given the code below, we use the [`map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) function to take an array of `numbers` and double their values. We assign the new array returned by `map()` to the variable `doubled` and log it:
+Jika melihat kode di bawah, kita menggunakan fungsi [`map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) untuk mengambil array `numbers` dan menggandakan nilainya.
+Kita akan menaruh array baru yang dikembalikan oleh `map()` ke dalam sebuah variabel `doubled` dan me-log-nya:
```

Comment:
kata log, mestinya pakai italic

Reply:
ok.. thank you @mazipan  i'll update tonight and make sure everything is fixed.. 
thanks again (y)

Reply:
Make sure, read #2 first.
Make it consistent!